src/deepsparse/server/protocol.py

```diff
@@ -142,17 +139,14 @@ class CompletionRequest(BaseModel):
     n: Optional[int] = 1
     stream: Optional[bool] = False
     logprobs: Optional[int] = None
-    echo: Optional[bool] = False
     stop: Optional[Union[str, List[str]]] = Field(default_factory=list)
     presence_penalty: Optional[float] = 0.0
     frequency_penalty: Optional[float] = 0.0
-    best_of: Optional[int] = None
     logit_bias: Optional[Dict[str, float]] = None
     user: Optional[str] = None
     # Additional parameters
     top_k: Optional[int] = -1
     ignore_eos: Optional[bool] = False
-    use_beam_search: Optional[bool] = False
```

Comment:
nit: i'm not sure if the error will be informative for the user if these arguments are provided and the parsing fails, rather than having explicit errors that they aren't supported once we parse the args in the request function

Reply:
I usually look at api specs to see what is supported (at least when using other 3rd party tools in the past). I guess that would be openai docs which would be misleading, unless we had specific ones for deepsparse?